pom.xml

```diff
@@ -78,7 +78,7 @@ THE SOFTWARE.
     <properties>
         <revision>2.0.8</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.361.4</jenkins.version>
+        <jenkins.version>2.420-rc34091.108b_c22ede78</jenkins.version> <!-- TODO: https://github.com/jenkinsci/jenkins/pull/8310 -->
```

Comment:
I guess this should be `2.414` to be aligned with the BOM version?

Reply:
sounds good, I will go back to 2.414, but there still will be the failing test. Should I add an '@Ignore` with a comment on the related jenkins ticket then?

Reply:
> Should I add an '@Ignore` with a comment on the related jenkins ticket then?

AFAIUI the intent of `testSshKeysCredentialsIdRemainsUnchangedAfterUpdatingOtherFields` is to check the credentials are not changing after editing it, so we could also slightly adjust the test to change other field instead of the cloud name. That way we do not have to ignore the test or block the release. I would add also a comment with the link to the core issue so we restore/add the original test when done.

Reply:
test updated in 3ae03ee064c62110cdefc1d2d209ab325fedc6de